### PR TITLE
Real users

### PR DESCRIPTION
**Pull request by seakelps**

### CODE REVIEW

File: templates/registration/login.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            <div>
+                <td>{{ form.password.label_tag }}</td>
+                <td>{{ form.password }}</td>
+            </div>
+
+            <div>
+              <input type="submit" value="login" class="btn"/>
+              <input type="hidden" name="next" value="{{ next }}" />
+            </div>
+            </form>
```
Comment (AlexRiina) [inline templates/registration/login.html]:
```
<form...>
{% csrf_token %}
{{ form.as_p }}
<input type="submit" value="login" class="btn"/>
</form>
```

is probably fine. it'd fix the `<td>` in the middle of the page and add the field.errors that are missing

File: templates/registration/password_reset_confirm.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                        <tr>
+                            <td>{{ form.new_password2.errors }}
+                                <label for="id_new_password2">Confirm password:</label></td>
+                            <td>{{ form.new_password2 }}</td>
+                        </tr>
+                        <tr>
+                            <td></td>
+                            <td><input type="submit" value="Change my password" / class="btn"></td>
+                        </tr>
+                    </table>
```
Comment (AlexRiina) [inline templates/registration/password_reset_confirm.html]: I think I'd probably do `{{ form.as_p }}` here too for consistency. not sure why the docs went through this manually. Also random / in the middle of your submit button

File: templates/registration/password_reset_form.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+{% block content %}
+
+<div class="container">
+
+    <div class="row">
+        <div class="col-md-12">
+            <form action="" method="post">{% csrf_token %}
+                {% if form.email.errors %} {{ form.email.errors }} {% endif %}
+                    <p>email address: {{ form.email }}</p>
```
Comment (AlexRiina) [inline templates/registration/password_reset_form.html]: same here. I don't think there are any extra fields, but `{{ form.as_p }}` will handle the errors, label, and field

File: templates/registration/login.html (hunk 1 above)
Comment (seakelps) [reply templates/registration/login.html]: I blame the mozilla docs!

File: templates/registration/password_reset_form.html (hunk 3 above)
Comment (seakelps) [reply templates/registration/password_reset_form.html]: adding changing email stuff to "{{ form.as_p }}" to our "after the election" list

**Pull request merged**